website/source/docs/guides/deployment.html.md

```diff
@@ -1,12 +1,13 @@
 ---
 layout: "docs"
-page_title: "Production Deployment"
-sidebar_current: "docs-guides-deployment"
+page_title: "Consul Reference Architecture"
+sidebar_current: "docs-guides-reference-architecture"
 description: |-
-  Best practice approaches for Consul production deployments.
+  This document provides recommended practices and a reference architecture for
+  HashiCorp Consul implementations.
 ---
 
-# Consul Production Deployment Guide
+# Consul Reference Architecture
```

Comment:
Just wanted to note that I'm not 100% sure "Reference Architecture" is the right phrasing here for this guide. We may want to consider other options. Once we finalize a name we should also align the names of the guides with the filenames, currently having two at `deployment*.html` may be kind of confusing to users / search engines.